gossipd/make_gossip.c

```diff
@@ -0,0 +1,456 @@
+/* Routines to make our own gossip messages */
```

Comment:
No, these are the routines which generate the gossip for our node and channels.  But the name "generate" is used otherwise in the source... agree it could be better.